CCS/MatrixOps/ccsmatops.pd

```diff
@@ -159,16 +156,13 @@ pp_def
   ('ccs_matmult2d_zdd',
    Pars => ("\n    "
             .join("\n    ",
-                  "$INDX ixa(Two,NnzA); nza(NnzA);", ## a(M,N) (M~i, N~x)
+                  "$INDX ixa(Two=2,NnzA); nza(NnzA);", ## a(M,N) (M~i, N~x)
```

Comment:
amazing ... wonderful ... thanks!

Reply:
That's been in PDL for at least 11 years!